example_project/measures/BuildResidentialModel/measure.rb

```diff
@@ -72,7 +72,12 @@ def arguments(model)
     arg.setDescription('The name of the folder containing a custom HPXML file, relative to the xml_building folder.')
     args << arg
 
-    measures_dir = File.absolute_path(File.join(File.dirname(__FILE__), '../../resources/hpxml-measures'))
+    arg = OpenStudio::Measure::OSArgument.makeIntegerArgument('building_id', false)
```

Comment:
If we copy the user-specified buildstock CSV file to a static location prior to this measure, I don't think we'd need to expose a buildstock CSV filepath argument here.

Reply:
Decided to expose a buildstock CSV filepath argument. Then we can remove all the lib folder business.